src/d3d12/D3D12_Functions.cpp

```diff
@@ -484,7 +482,7 @@ void D3D12::PrepareUpdate()
 
     std::lock_guard _(m_imguiLock);
 
-    ImGui_ImplWin32_NewFrame(m_outSize);
+    ImGui_ImplWin32_NewFrame();
```

Comment:
same for size, otherwise breaks with DPI scaling on anything other than 100%

Reply:
Newer versions of ImGui have some level of DPI awareness in the Win32 impl. I can add this back easily though if we need to pass it back. I have DPI scaling on my computer and it seems to work fine without this. Similar to how it was looking in the current version.

Reply:
as long as ImGui seem to report correct coords I believe this should be fine then